WordPress/WordPressTest/MediaPicker/MediaLibraryPickerDataSourceTests.swift

```diff
@@ -10,7 +10,7 @@ class MediaLibraryPickerDataSourceTests: XCTestCase {
     fileprivate var blog: Blog!
     fileprivate var post: Post!
 
-    fileprivate func newMediaImage() -> Media? {
+    fileprivate func newImageMedia() -> Media? {
```

Comment:
It looks like the `newImageMedia` and `newVideoMedia` methods share most of their code. Perhaps refactor them into something like this? (and I'd personally push them down to the bottom of the implementation, as they're private helper methods)

```
// MARK: Private helpers

fileprivate func newMedia(forResource resource: String, withExtension ext: String) -> Media? {
    var newMedia: Media?
    guard let url = Bundle(for: type(of: self)).url(forResource: resource, withExtension: ext),
        let mediaService = MediaService(managedObjectContext: context) else {
            XCTFail("Pre condition to create media service failed")
            return nil
    }
    let expect = self.expectation(description: "Media should be create with success")
    mediaService.createMedia(url: url, forPost: post.objectID, thumbnailCallback: { (url) in
    }, completion: { (media, error) in
        expect.fulfill()
        if let _ = error {
            XCTFail("Media should be created without error")
            return
        }
        newMedia = media
    })
    self.waitForExpectations(timeout: 5, handler: nil)
    return newMedia
}

fileprivate func newImageMedia() -> Media? {
    return newMedia(forResource: "test-image", withExtension: "jpg")
}

fileprivate func newVideoMedia() -> Media? {
    return newMedia(forResource: "test-video", withExtension: "mp4")
}
```